core/platform/cloud_translate/cloud_translate_services_test.py

```diff
@@ -0,0 +1,59 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for cloud_translate_services."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+from core.platform.cloud_translate import cloud_translate_services
+from core.tests import test_utils
+
+
+class CloudTranslateServicesUnitTests(test_utils.TestBase):
+    """Tests for cloud_translate_services."""
+
+    def test_translate_text_with_invalid_source_language_raises_error(self):
+        with self.assertRaisesRegexp(
+            ValueError, 'invalid language code: invalid'):
+
```

Comment:
Nit: New line not necessary here. Ditto below.

Reply:
I removed the newline.
I had added it because the indent from the line above makes it hard to tell where the `with` statement ends, is there a better solution for that?